### ISSUE TITLE
Several seconds of delay before showing output of winget upgrade/list (Windows 11)

### ISSUE THREAD

**Issue by adrianghc** [microsoft/winget-cli#1387]
# Brief description of your issue

Ever since I upgraded to Windows 11, I've found that running `winget upgrade` or `winget list` has a delay of several seconds before the command's output is shown. This happened almost instantaneously on Windows 10. My settings file only enables the Microsoft Store experiment, but this problem also appears when that is disabled.

https://user-images.githubusercontent.com/11506007/130304313-6d2a9824-1731-4496-9997-0e0196d6db6e.mp4

# Steps to reproduce

Run `winget upgrade` or `winget list` in a command line.

# Expected behavior

After winget's rotating loading indicator disappears, the results of `winget upgrade` should display almost immediately.

# Actual behavior

There is a delay of several seconds where the CLI's blinking cursor appears instead of the command's output.

# Environment

```none
Windows Package Manager v1.0.11694
Windows: Windows.Desktop v10.0.22000.160
Package: Microsoft.DesktopAppInstaller v1.15.11694.0

Any other software?
```

**Comment by denelon** [comment]
@adrianghc the experimental store source is now REST based and behaves quite a bit differently than the PreIndexed package we used in the past. We do have some performance work that needs to be done here. Do you see the same results if you disable the store source (temporarily to test)?

**Comment by adrianghc** [comment]
@denelon Yes, as I say in the issue, this happens also when the store source is disabled.

**Comment by denelon** [comment]
@adrianghc my bad 🤦‍♂️ I read it too quickly. We do have an Issue to work on performance in either case. I was just trying to make sure you weren't seeing something different between the two.

The MSStore Source will not be needed after the next release. If you aren't referencing the MSStore source as a "PreIndexed" package when you run `winget source export` you wont need it. We are planning to wait for a release or two before we automatically remove it (assuming you don't have a reference to the PreIndexed package).

I've shared the output from my system below for reference. It's the last value.

```
PS C:\Users\denelon\GitHub\winget-pkgs\Tools> winget source export
{"Arg":"https://winget.azureedge.net/cache","Data":"Microsoft.Winget.Source_8wekyb3d8bbwe","Identifier":"Microsoft.Winget.Source_8wekyb3d8bbwe","Name":"winget","Type":"Microsoft.PreIndexed.Package"}
{"Arg":"https://storeedgefd.dsx.mp.microsoft.com/v9.0","Data":"","Identifier":"StoreEdgeFD","Name":"storepreview","Type":"Microsoft.Rest"}
```

**Comment by adrianghc** [comment]
So it turns out that while the experimental store source was disabled through settings, the actual newstore source was not. Removing the newstore source restores the winget commands to their former speed.

**Comment by denelon** [comment]
@adrianghc,

We've published the v1.1 [release candidate](https://github.com/microsoft/winget-cli/releases/tag/v1.1.12653).

If you run `winget source reset` any "extra" sources will be removed and the default sources will be added back.

**Comment by zotabee** [comment]
Fixed! Thanks a lot!
